web/src/torrent.js

```diff
@@ -388,7 +388,11 @@ export class Torrent extends EventTarget {
 
     // maybe filter by text...
     if (pass && search) {
-      pass = this.getCollatedName().includes(search.toLowerCase());
+      const searchLower = search.toLowerCase();
+      const nameMatch = this.getCollatedName().includes(searchLower);
+      const location = this.getDownloadDir() ? this.getDownloadDir().toLowerCase() : '';
```

Comment:
I think [operator chaining](https://caniuse.com/mdn-javascript_operators_optional_chaining) + [nullish coalescing](https://caniuse.com/mdn-javascript_operators_nullish_coalescing) should be safe to use in 2025?


```suggestion
      const location = this.getDownloadDir()?.toLowerCase() ?? '';
```